api/flaskr/routes/auth.py

```diff
@@ -0,0 +1,119 @@
+from flask import Blueprint, request, url_for, redirect, session, current_app, jsonify
+from werkzeug.wrappers import Response
+from flask_cors import cross_origin
+from flask_jwt_extended import (
+    JWTManager, create_access_token, create_refresh_token,
+    jwt_required, get_jwt_identity, jwt_refresh_token_required,
+    get_raw_jwt
+)
+
+from flask_bcrypt import Bcrypt
+import datetime
+
+from ..db import mongo
+
+bcrypt = Bcrypt(current_app)
+jwt = JWTManager(current_app)
+blacklist = set()
+
+auth_route = Blueprint('auth_route', __name__)
+
+
+@auth_route.route('/signup', methods=(['POST']))
+def signup():
+    # Receive the form info
+    username = request.form['username']
+    password = request.form['password']
+    email = request.form['email']
+    name = request.form['name']
+    surname = request.form['surname']
+
+    # Get the users collection
+    users = mongo.userInformation.db.users
+
+    # Check for duplicates
+    existing_user = users.find_one({'username': username})
+    existing_email = users.find_one({'email': email})
+
+    if (existing_user is None):
+        if (existing_email is None):
+            # encrypt the password
+            pw_hash = bcrypt.generate_password_hash(
+                password).decode('utf-8')
+            users.insert(
+                {'username': username, 'email': email, 'password': pw_hash, 'name': name, 'surname': surname})
+            session['username'] = username
+
+            return redirect(url_for('index'))
+        else:
+            return Response('That email already exists!', status=403)
+    else:
+        return Response('That username already exists!', status=403)
+
+
+@auth_route.route('/login', methods=(['POST']))
+def login():
+    # Receive the form info
+    username = request.form['username']
+    password = request.form['password']
+    # email = request.form['email']
+
+    # Get the users collection
+    users = mongo.userInformation.db.users
+
+    login_user = users.find_one({'username': username})
+    if login_user:
+        if bcrypt.check_password_hash(login_user['password'], password):
+            session['username'] = username
+            expires = datetime.timedelta(hours=1)
+            tokens = {
+                'access_token': create_access_token(identity=username, expires_delta=False),
+                'refresh_token': create_refresh_token(identity=username)
+            }
+            return jsonify(tokens), 200
```

Comment:
is it possible to edit this line as ?
```python
return Response(tokens, status=200)
```
or 
```python
return Response(jsonify(tokens), status=200)
```
I'm not sure which one of the two lines of code would work, thus I suggested both. I think you get the idea 😄 

I also think returning a Response object in each of the routes would make our code more consistent.